sdk/aqueduct/flow.py

```diff
@@ -85,7 +85,8 @@ def _construct_flow_run(
                 continue
 
             dag.update_operator_spec(
-                param_artifact.name,  # this works because the parameter op and artifact currently share the same name.
+                # this works because the parameter op and artifact currently share the same name.
```

Comment:
was this a linter change change? If not I feel like same line looks better :P

Reply:
Yes, this is a linter change.